geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/DestroyConnectionCommand.java

```diff
@@ -54,24 +54,18 @@ public Result destroyConnection(@CliOption(key = DESTROY_CONNECTION__NAME, manda
     // action
     List<CliFunctionResult> results =
         executeAndGetFunctionResult(new DestroyConnectionFunction(), name, targetMembers);
-
-    boolean persisted = false;
-    ClusterConfigurationService ccService = getConfigurationService();
-
-    if (ccService != null && results.stream().filter(CliFunctionResult::isSuccessful).count() > 0) {
-      ConnectorService service =
-          ccService.getCustomCacheElement("cluster", "connector-service", ConnectorService.class);
-      if (service != null) {
-        ConnectorService.Connection conn = CacheElement.findElement(service.getConnection(), name);
-        service.getConnection().remove(conn);
-        ccService.saveCustomCacheElement("cluster", service);
-        persisted = true;
-      }
-    }
-
     CommandResult commandResult = ResultBuilder.buildResult(results);
-    commandResult.setCommandPersisted(persisted);
-
+    CacheElement element = (CacheElement) () -> name;
```

Comment:
Using an anonymous class here is confusing.  I don't know if partially filling out
```
    ConnectorService elementToRemove = new ConnectorService();
    elementToRemove.setName(name);
```
is any less confusing, though.

This comment could be repeated for every `Destroy*Command`.